Readme.md

```diff
@@ -48,13 +48,24 @@ default: false
 Whether to show [Brotli](https://www.wikiwand.com/en/Brotli) size or not
 
 #### showBeforeSizes
-type: `boolean`
+
+type: `boolean` or `"lastBuild"`
```

Comment:
I think the values shoul be `build`, `release` and `none`

Reply:
Ok, I've applied.

Reply:
Thank you.

Reply:
Sweet, thanks! Do you think you could put out a release too?

Reply:
after version bump, 2 tests seem to be failing on master. Seems to be something related to checking the size from the last released bundle (which has changed now). I just looked to the file and couldn't get time to debug. I will have a look later today. If you find something, just ping me here. I will do a beta release just after that :)

Reply:
Hey @brettz9 , also will you like to be added as a collaborator to this repo?

Reply:
I've filed #69 to address this test failure (though note sure you want me to act on my follow-up comment there instead: https://github.com/ritz078/rollup-plugin-filesize/pull/69#issuecomment-620975295 ).

Re: being added as a collaborator, sure. I can't guarantee I could always assist with the responsibilities, but I'd be happy to help as I have time/am so inspired. Just let me know your policy--e.g., make PRs for new features and directly commit for obvious fixes and trivial chores?

Reply:
Continuing the conversation here: https://github.com/ritz078/rollup-plugin-filesize/pull/69#issuecomment-620984806

Reply:
Hi @brettz9,

I have sent you an invite. Regarding the policies, I think it makes sense to always make a PR and wait for the tests to pass before merging into master. If it's a quick fix, there's no need for a review. If there's a behaviour change or a new feature, it always helps to have a review. WDYT?

Reply:
Thanks! And SGTM!